### PR TITLE
Kevin Marker

### PR DESCRIPTION
**Pull request by kmarker1101**

### CODE REVIEW

**Review by johnoro — COMMENTED** [review]
Great work!

File: Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js [hunk 1]
```diff
@@ -1,2 +1,18 @@
 import React from 'react';
+import Moment from 'moment';
 import './Header.css';
+
+
+const HeaderTitle = () => {
+    //const dateToFormat = Moment().format("DD MMM");
```
[the remark this answers is not in the document]
Comment (kmarker1101) [reply Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js]: Done

File: Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Footer.css';
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faEnvelope} from '@fortawesome/free-solid-svg-icons'
+import {faHeart} from '@fortawesome/free-solid-svg-icons'
+import {faComment} from '@fortawesome/free-solid-svg-icons'
+import {faSync} from '@fortawesome/free-solid-svg-icons'
```
[the remark this answers is not in the document]
Comment (kmarker1101) [reply Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js]: done

File: Day-I-Social-Card/social-card/package.json [hunk 3]
```diff
@@ -3,6 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.8",
+    "@fortawesome/free-solid-svg-icons": "^5.5.0",
+    "@fortawesome/react-fontawesome": "^0.1.3",
+    "i": "^0.3.6",
```
[the remark this answers is not in the document]
Comment (kmarker1101) [reply Day-I-Social-Card/social-card/package.json]: done

File: Day-I-Social-Card/social-card/package.json [hunk 4]
```diff
@@ -12,5 +18,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.5.0"
```
[the remark this answers is not in the document]
Comment (kmarker1101) [reply Day-I-Social-Card/social-card/package.json]: done

**Review by johnoro — COMMENTED** [review]
Great job!

**Pull request closed without merging**